### ISSUE TITLE
[New Feature] Badge external

### ISSUE THREAD

**Issue by 1chooo** [JacobLinCool/LeetCode-Stats-Card#129]
Hi Thanks for this amazing project at first.

I would like to suggest the new feature that we can change the `ext=all_badges` or maybe we can rename it later! With this feature we can display all the badges we have earned in the LeetCode Stats Card.

If yes, maybe we can have the further discuss, and please let me know if I can help you!

<img width="854" alt="Screenshot 2024-07-23 at 12 30 06 AM" src="https://github.com/user-attachments/assets/dda21c0b-68b1-4a89-809c-c9b4289c4fca">

**Comment by JacobLinCool** [comment]
Displaying multiple badges is a great feature, but showing "all" badges might have some issues. For example, some users (like this one: https://leetcode.com/u/AoK_Blast/) have an excessive number of badges.

I suggest showing a maximum of 4 or 5 recent badges or providing an option for users to select which badges they want to display.

I might be able to try this later, or I'd be happy to review if you'd like to submit a PR.

**Comment by 1chooo** [comment]
> Displaying multiple badges is a great feature, but showing "all" badges might have some issues. For example, some users (like this one: https://leetcode.com/u/AoK_Blast/) have an excessive number of badges.
> 
> I suggest showing a maximum of 4 or 5 recent badges or providing an option for users to select which badges they want to display.
> 
> I might be able to try this later, or I'd be happy to review if you'd like to submit a PR.

Yeah, your concern is right, the maximum of 4 or 5 recent badges is a great advise! I could have spent some time to research how to achieve it!
